backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/document/model/impl/AbstractElasticsearchIndexSchemaCollector.java

```diff
@@ -36,9 +37,10 @@ public String toString() {
 	public abstract ElasticsearchIndexSchemaElement withContext(IndexSchemaNestingContext context);
 
 	@Override
-	public ObjectFieldIndexSchemaCollector objectField(String relativeName) {
+	public ObjectFieldIndexSchemaCollector objectField(String relativeName, ObjectFieldStorage storage) {
 		IndexSchemaObjectPropertyNodeBuilder nestedNodeBuilder =
```

Comment:
The variable name should probably be changed.

Reply:
Will do.